code-challenges.test.js

```diff
@@ -12,49 +13,113 @@
 
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
+// Pseudocode
+// input: variable which is a number (the length of the array)
+// output: an array that is the length of the input containing the numbers of the Fibonacci sequence.
+// const fibLength1 = 6
+    // Expected output: [1, 1, 2, 3, 5, 8]
+// const fibLength2 = 10
+    // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+// Process: I will create a function called fibArray that will take in the variable and will output an array that is the length of the input containing the numbers of the Fibonacci sequence. I will create a for loop that create the array that then pushes the values into an empty array. I will set the first two values of the array at [1, 1] so that it starts with those first two values.
+
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
+describe("fibFunction", () => {
+  it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence", () => {
+    const fibLength1 = 6
```

Comment:
GREAT job hoisting your given variables into your Jest tests! This is a really good practice.